apps/sms/js/thread_list_ui.js

```diff
@@ -421,8 +421,11 @@ var ThreadListUI = {
       });
     }
 
-    return Utils.confirm(
-      'deleteThreads-confirmation2', null,
+    var selectNumber = this.selectionHandler.selectedCount;
+
+    return Utils.confirm({
+      id: 'deleteThreads-confirmation-message',
+      args: { n: selectNumber } }, null,
```

Comment:
And you can just set the number directly instead of declare it:
`args: { n: this.selectionHandler.selectedCount }`